cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -11,7 +11,7 @@
     MaxNumberOfNodesForStake = 28
     NodesToSelectInAuction = 28
     UnJailValue = "2500000000000000000" #0.1% of genesis node price
-    ActivateBLSPubKeyMessageVerification = false #TODO set to true before mainnet
+    ActivateBLSPubKeyMessageVerification = true
```

Comment:
It will be activated when functionality pf staking (system smart contracts) will be also activated. For now this flag has no importance as staking is also disabled. Anyway, thanks for this reminder.